packages/blade/src/components/Typography/Text/Text.tsx

```diff
@@ -92,9 +89,7 @@ const getTextProps = <T extends { variant: TextVariant }>({
 const StyledText = styled(BaseText)(({ truncateAfterLines }) => {
   if (truncateAfterLines) {
     if (getPlatformType() === 'react-native') {
-      return {
-        numberOfLines: truncateAfterLines,
-      };
+      return null;
```

Comment:
Why are we returning null here instead?

Reply:
since react native doesn't accept `numberOfLines` as a style property. Before this change, react native was throwing a warning for this. We're passing `numberOfLines` as a prop to react native's Text